client/src/components/search/searchBar/searchbar.css

```diff
@@ -144,19 +106,50 @@
   font-weight: bold;
 }
 
-/* Show only the first 5 hits on mobile */
-.ais-Hits-list .ais-Hits-item:nth-child(n + 6) {
-  display: none;
+@media (min-width: 380px) {
+  .fcc_searchBar .ais-Hits {
+    width: calc(100vw - 30px);
+    left: 15px;
+  }
+  .ais-SearchBox-input {
+    width: calc(100vw - 30px);
+  }
+  .fcc_searchBar .ais-SearchBox-form {
+    display: flex;
+    position: absolute;
+    top: var(--header-height);
+    right: 15px;
+  }
 }
 
-/* Ensure the dropdown footer is always visible */
-.ais-Hits-list .ais-Hits-item:nth-child(9) {
-  display: block;
+@media (min-width: 700px) {
+  .ais-SearchBox-input {
+    width: 100%;
+  }
+  #fcc_instantsearch {
```

Comment:
It looks like there's a hydration issue with all the `#fcc_instantsearch` selectors (this is from .dev):
![Jump](https://user-images.githubusercontent.com/15801806/65783513-3d160f80-e150-11e9-936e-3d997aea89a2.gif)
since the id is set by js.  Could all the id selectors be replaced using classes that start off present in the html?

Reply:
Thank you for catching this. I'll work on this and pull in @ahmadabdolsaheb for further review.